common/libexec/wwinit/10-database.init

```diff
@@ -111,24 +122,14 @@ fi
 
 if [ -n "$DBUSER" -a "$DBUSER" != "root" ]; then
     wwprint "Updating database permissions for base users"
-    if echo "GRANT SELECT ON $DBNAME.* TO $DBUSER@localhost IDENTIFIED BY '$DBPASS';" | mysql $DBNAME; then
-        reply_success
-    else
-        reply_error
-    fi
-fi
-
-if [ -n "$DBROOTUSER" -a "$DBROOTUSER" != "root" ]; then
-    wwprint "Updating database permissions for root user"
-    if echo "GRANT ALL PRIVILEGES ON $DBNAME.* TO $DBROOTUSER@localhost IDENTIFIED BY '$DBROOTPASS';" | mysql $DBNAME; then
+    if echo "GRANT ALL PRIVILEGES ON $DBNAME.* TO $DBUSER IDENTIFIED BY '$DBPASS';" | mysql $DBNAME; then
```

Comment:
Because the database may be on a different machine. Actually, the script should be modified to read the host from the configuration file. I will try to correct this.